### ISSUE TITLE
ERROR: Could not find a version that satisfies the requirement torch==1.2.0

### ISSUE THREAD

**Issue by MohitShridhar** [alfworld/alfworld#20]
```
ERROR: Could not find a version that satisfies the requirement torch==1.2.0 (from alfworld) (from versions: 0.1.2, 0.1.2.post1, 0.1.2.post2, 1.4.0, 1.5.0, 1.5.1, 1.6.0, 1.7.0, 1.7.1, 1.8.0, 1.8.1)
ERROR: No matching distribution found for torch==1.2.0 (from alfworld)
```

Need to add `-f https://download.pytorch.org/whl/torch_stable.html`

**Comment by Hua-rookie** [comment]
It still did not work after adding that.

**Comment by MarcCote** [comment]
Can you share the error message. Also, last time I installed ALFWorld I needed to use Python 3.6.

**Comment by Hua-rookie** [comment]
oh, it's solved after I tried Python 3.6. thanks a lot!
